### PR TITLE
DOCS-3257 Synthetics API Tests Assertions Section Edits

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Updates assertions on most API Test types and screenshots for assertion sections.

### Motivation

DOCS-3257

### Preview

https://docs-staging.datadoghq.com/alai97/synthetics-api-tests-assertions-doc-update/synthetics/api_tests/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by buraizu — CHANGES_REQUESTED** [review]
Hey Austin, could you add a bit more description to the alt text for the new images?

**Review by HugoPct — APPROVED** [review]
Looks good to me!

File: content/en/synthetics/api_tests/dns_tests.md [hunk 1]
```diff
@@ -50,12 +50,12 @@ Assertions define what an expected test result is. After you click **Test URL**,
...
-| every record        | of type A, of type AAAA, of type MX, of type TXT, of type CNAME | `is`, `contains`, <br> `matches`, `does not match` | _String_ <br> _[Regex][5]_ |
-| at least one record | of type A, of type AAAA, of type MX, of type TXT, of type CNAME | `is`, `contains`, <br> `matches`, `does not match` | _String_ <br> _[Regex][5]_ |
+| every available record        | of type A, of type AAAA, of type CNAME, of type MX, of type NS, of type TXT | `is`, `contains`, <br> `matches`, `does not match` | _String_ <br> _[Regex][5]_ |
+| at least one record | of type A, of type AAAA, of type CNAME, of type MX, of type NS, of type TXT | `is`, `contains`, <br> `matches`, `does not match` | _String_ <br> _[Regex][5]_ |
 
 You can create up to 20 assertions per API test by clicking **New Assertion** or by clicking directly on the response preview:
 
-{{< img src="synthetics/api_tests/assertions.png" alt="Define assertions for your DNS test" style="width:90%;" >}}
+{{< img src="synthetics/api_tests/assertions_dns.png" alt="Define assertions for your DNS test" style="width:90%;" >}}
```
Comment (alai97) [inline content/en/synthetics/api_tests/dns_tests.md:58]:
```suggestion
{{< img src="synthetics/api_tests/assertions_dns.png" alt="Define assertions for your DNS test to succeed or fail on" style="width:90%;" >}}
```

File: content/en/synthetics/api_tests/http_tests.md [hunk 2]
```diff
@@ -126,7 +126,7 @@ HTTP tests can decompress bodies with the following `content-encoding` headers:
 
 You can create up to 20 assertions per API test by clicking **New Assertion** or by clicking directly on the response preview:
 
-{{< img src="synthetics/api_tests/assertions.png" alt="Define assertions for your HTTP test" style="width:90%;" >}}
+{{< img src="synthetics/api_tests/assertions_http.png" alt="Define assertions for your HTTP test" style="width:90%;" >}}
```
Comment (alai97) [inline content/en/synthetics/api_tests/http_tests.md:129]:
```suggestion
{{< img src="synthetics/api_tests/assertions_http.png" alt="Define assertions for your HTTP test to succeed or fail on" style="width:90%;" >}}
```

File: content/en/synthetics/api_tests/ssl_tests.md [hunk 3]
```diff
@@ -64,7 +64,7 @@ Assertions define what an expected test result is. After you click **Test URL**,
 
 You can create up to 20 assertions per API test by clicking **New Assertion** or by clicking directly on the response preview:
 
-{{< img src="synthetics/api_tests/assertions.png" alt="Define assertions for your SSL test" style="width:90%;" >}}
+{{< img src="synthetics/api_tests/assertions_ssl.png" alt="Define assertions for your SSL test" style="width:90%;" >}}
```
Comment (alai97) [inline content/en/synthetics/api_tests/ssl_tests.md:67]:
```suggestion
{{< img src="synthetics/api_tests/assertions_ssl.png" alt="Define assertions for your SSL test to succeed or fail on" style="width:90%;" >}}
```

File: content/en/synthetics/api_tests/tcp_tests.md [hunk 4]
```diff
@@ -46,10 +46,12 @@ Assertions define what an expected test result is. When you click **Test URL**,
 | Type          | Operator                                                                | Value type     |
 |---------------|-------------------------------------------------------------------------|----------------|
 | response time | `is less than`                                                          | _Integer (ms)_ |
+| network hops    | `is less than`, `is less than or equal`, `is`, `is more than`, `is more than or equal` | _integer_        |
+| connection | `is`                                                          | `established`, `refused`, `timeout` |
 
 You can create up to 20 assertions per API test by clicking **New Assertion** or by clicking directly on the response preview:
 
-{{< img src="synthetics/api_tests/assertions.png" alt="Define assertions for your TCP test" style="width:90%;" >}}
+{{< img src="synthetics/api_tests/assertions_tcp.png" alt="Define assertions for your TCP test" style="width:90%;" >}}
```
Comment (alai97) [inline content/en/synthetics/api_tests/tcp_tests.md:54]:
```suggestion
{{< img src="synthetics/api_tests/assertions_tcp.png" alt="Define assertions for your TCP test to succeed or fail on" style="width:90%;" >}}
```

**Review by buraizu — APPROVED** [review]

**Pull request merged**